shpc/main/modules/tcl.py

```diff
@@ -12,7 +12,10 @@ def __init__(self, **kwargs):
         An Lmod client generates an lmod recipe for install
         """
         super(Client, self).__init__(**kwargs)
+        # The extension is technically not required, but we still set one for clarity
         self.module_extension = "tcl"
+        # Except for symlink, since the goal is to have short names
+        self.symlink_extension = ""
```

Comment:
Oh I see (my question before). Just to clarify - it's required for lua?

Reply:
Yes. Lmod requires the Lua modules to end with `.lua`. TCL doesn't have such requirements

Reply:
Gotcha, thanks!